### PR TITLE
feat: add ProfileBuilderNew for java

### PR DESCRIPTION
**Pull request by ddemonaz89**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +175,7 @@ Creates feeder capable of retrieving secret data from HC Vault
   val vaultFeeder = VaultFeeder(vaultUrl, secretPath, roleId, secretId, keys)
 ```
 
+Kotlin example:
```
Comment (red-bashmak) [inline README.md:178]: Perhaps this line should be higher up.

Comment (ddemonaz89) [reply README.md]: fix

File: README.md [hunk 2]
```diff
@@ -161,10 +161,12 @@ Creates feeder capable of retrieving secret data from HC Vault
...
     - keys - list of keys you are willing to retrieve from vault
 
+Scala example:
 ```scala
   val vaultFeeder = VaultFeeder(vaultUrl, secretPath, roleId, secretId, keys)
 ```
 
+Java example:
 ```Java
   Iterator<Map<String, Object>> vaultFeeder = VaultFeeder(vaultUrl, secretPath, roleId, secretId, keys);
```
Comment (red-bashmak) [inline README.md:171]: New versions of Java allow this shorter form: `var vaultFeeder = ...`. More informartion [here](https://openjdk.org/jeps/286)

File: src/test/java/ru/tinkoff/gatling/javaapi/JavaProfileTest.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package ru.tinkoff.gatling.javaapi;
+
+import io.gatling.javaapi.core.ScenarioBuilder;
+import ru.tinkoff.gatling.javaapi.profile.ProfileBuilderNew;
+
+
+public class JavaProfileTest {
+    ScenarioBuilder scn = ProfileBuilderNew
+            .buildFromYaml("perf/profiles.yml")
```
Comment (red-bashmak) [inline src/test/java/ru/tinkoff/gatling/javaapi/JavaProfileTest.java:9]: What will happen if here be a path to a file that doesn't exist?

**Review by red-bashmak — CHANGES_REQUESTED** [review]
Some fixes needed. More in comments

**Review by jigarkhwar — APPROVED** [review]

**Pull request merged**